server/app.js

```diff
@@ -1,17 +1,13 @@
 import express from 'express';
```

Comment:
Looks like its just a couple files, mostly the controllers and Sequelize code. converting it to import shouldn't be that difficult. I like the ES6 syntax a bit better, so its my preference.